### ISSUE TITLE
Need to test SSL builds and functionality

### ISSUE THREAD

**Issue by SpamapS** [gearman/gearmand#94]
If we're going to have crypto, we need to make sure it works. And we need to make sure it works with both wolfSSL and OpenSSL.

**Comment by SpamapS** [comment]
I'd like to see two different types of tests.

1) something in the tests suite that will ensure an SSL enabled libgearman can connect to an SSL enabled gearmand.

2) Something that tests some other native language binding (python maybe) and ensures their SSL can connect to a python enabled gearmand.

**Comment by p-alik** [comment]
let me see if [gearman](https://github.com/gearman/gearmand/blob/master/bin/gearman.cc) cli could be used for the purpose

**Comment by esabol** [comment]
I think this issue was addressed in PR #486 ? Do you agree, @p-alik? I think this issue is (mostly?) a duplicate of issue #56.

**Comment by p-alik** [comment]
Agreed — PR #486 added an explicit SSL connectivity test (`t/ssl`) to `make check`, which covers what's being asked for here. Closing as a duplicate of #56.
